package.json

```diff
@@ -40,8 +40,8 @@
   "dependencies": {
     "lodash": "^4.13.1",
     "moment": "^2.14.1",
-    "seamless-immutable": "^6.1.1",
+    "seamless-immutable": "^7.0.1",
```

Comment:
BitHound was complaining about the out of date version. I think we need to discuss BitHound usage since it seems like it's just going ignored so far.